### PR TITLE
org.opencontainers.ref.name -> org.opencontainers.image.ref.name

### PR DESCRIPTION
**Pull request by AkihiroSuda**
According to rc6 spec, the annotation was wrong https://github.com/opencontainers/image-spec/blob/v1.0.0-rc6/annotations.md

Signed-off-by: Akihiro Suda <suda.akihiro@lab.ntt.co.jp>

### CODE REVIEW

**Comment by AkihiroSuda** [comment]
CI failure seems unrelated
```
vendor/github.com/containers/storage/store.go:18:2: cannot find package "github.com/BurntSushi/toml" in any of:
	/home/travis/gopath/src/github.com/containers/image/vendor/github.com/BurntSushi/toml (vendor tree)
	/home/travis/.gimme/versions/go1.7.linux.amd64/src/github.com/BurntSushi/toml (from $GOROOT)
	/home/travis/gopath/src/github.com/BurntSushi/toml (from $GOPATH)
make: *** [test] Error 1
t
The command "make tools .gitvalidation validate test test-skopeo BUILDTAGS="$BUILDTAGS"" exited with 2.

````
https://travis-ci.org/containers/image/jobs/236321911

**Comment by runcom** [comment]
Thanks a lot for spotting this @AkihiroSuda - actually CI failure is related to the fact that we vendor latest containers/storage master and that introduced toml dep what we're not vendoring. Could you please add it to vendor.conf (do not run `vndr` then!) and also use a commit hash instead of master for containers/storage in `vendor.conf`? Thanks

**Comment by AkihiroSuda** [comment]
done

**Comment by runcom** [comment]
LGTM

**Comment by runcom** [comment]
@AkihiroSuda would you open a PR in skopeo to vendor c/image master over there to pull in this change? :angel:

File: oci/layout/oci_dest.go [hunk 1]
```diff
@@ -173,7 +173,7 @@ func (d *ociImageDestination) PutManifest(m []byte) error {
 	}
 
 	annotations := make(map[string]string)
-	annotations["org.opencontainers.ref.name"] = d.ref.tag
+	annotations["org.opencontainers.image.ref.name"] = d.ref.tag
```
Comment (runcom) [inline oci/layout/oci_dest.go]: actually, I'd rather use what's defined in oci image-spec itself, I'll follow up on this.

Comment (runcom) [inline oci/layout/oci_dest.go]: nevermind, they're not defined anywhere lol

**Comment by AkihiroSuda** [comment]
sure https://github.com/projectatomic/skopeo/pull/349

File: oci/layout/oci_dest.go (hunk 1 above)
Comment (AkihiroSuda) [inline oci/layout/oci_dest.go]: 😃 https://github.com/opencontainers/image-spec/pull/689

**Pull request merged**